docs/synapse/userguides/storm_ref_data_mod.rstorm

```diff
@@ -581,56 +742,97 @@ Remove Tags
+.. NOTE::
   
-  In addition, it is **essential** to understand how removing a tag at a given position in a tag tree affects other tags within that tree. Otherwise, tags may be improperly left in place ("orphaned") or inadvertently removed.
+  In Synapse 2.x, a tag must be deleted from the :ref:`gloss-layer` where the tag exists. Generally speaking, this means
+  you must be in the :ref:`gloss-view` where the relevant layer is the topmost (writeable) layer in order to delete the tag.
+
 
 **Syntax:**
 
 *<query>* **[ -#** *<tag>* ... **]**
 
 **Examples:**
 
-*Remove a leaf tag:*
+*Remove a leaf tag (i.e., the final or rightmost element of the tag):*
 
 
-.. storm-pre:: inet:ipv4 [ -#cno.infra.anon.tor ]
+.. storm-pre:: inet:ipv4 [ -#cno.infra.anon.tor.exit ]
 ::
 
-    <inet:ipv4> [ -#cno.infra.anon.tor ]
+    <inet:ipv4> [ -#cno.infra.anon.tor.exit ]
+
+
+*Remove a full tag (i.e., the entire tag):*
+
+.. storm-pre::  inet:ipv4 [ -#cno ]
+::
+  
+  <inet:ipv4> [ -#cno ]
 
 
 **Usage Notes:**
 
-- Tag deletions are performed on the output of a previous Storm query.
-- Storm will delete the specified tag(s) from all nodes in the current working set (i.e., all nodes resulting from Storm syntax to the left of the -#<tag> statement), **whether those nodes are within or outside of the brackets** unless :ref:`edit-parens` are used to limit the scope of the modifications.
-- Deleting a leaf tag deletes **only** the leaf tag from the node. For example, ``[ -#foo.bar.baz ]`` will delete the tag ``#foo.bar.baz`` but leave the tags ``#foo.bar`` and ``#foo`` on the node.
-- Deleting a non-leaf tag deletes that tag and **all tags below it in the tag hierarchy** from the node. For example, ``[ -#foo ]`` used on a node with tags ``#foo.bar.baz`` and ``#foo.hurr.derp`` will remove **all** of the following tags:
+- Deleting a leaf tag deletes **only** the leaf tag from the node. For example, ``[ -#foo.bar.baz ]`` will delete the tag
+  ``#foo.bar.baz`` but leave the tags ``#foo.bar`` and ``#foo`` on the node.
+- Deleting a non-leaf tag deletes that tag and **all tags below it in the tag hierarchy** from the node. For example,
+  ``[ -#foo ]`` used on a node with tags ``#foo.bar.baz`` and ``#foo.hurr.derp`` will remove **all** of the following tags:
 
   - ``#foo.bar.baz``
   - ``#foo.hurr.derp``
   - ``#foo.bar``
   - ``#foo.hurr``
   - ``#foo``
 
-- See the Storm :ref:`storm-tag-prune` command for recursive removal of tags (i.e., from a leaf tag up through parent tags that do not have other children).
+.. TIP::
+  
+  The Storm :ref:`storm-tag-prune` command can be used to recursively remove tags (i.e., from a leaf tag up through
+  parent tags that do not have other children).
+
+
+.. _tag-time-del:
+
+Remove Tag Timestamps
++++++++++++++++++++++
+
+To remove a tag timestamp from a tag, you must remove the tag element that contains the timestamp. The tag element
+can be re-added without the timestamp or property if needed.
```

Comment:
Got a leftover "or property" here that seems a bit out of place now that timestamps/properties are separate
```suggestion
can be re-added without the timestamp if needed.
```

Reply:
dangit